### PR TITLE
Fix/client details : Made client details page scrollable 

### LINKED ISSUE #86: Individual client details are not scrollable
Fix Client Details page to show client details as grid and now just in a column

Fix page to be scrollable

Image

### PR DESCRIPTION
**Pull request by inuoluwadunsimi**
Screenshot (64)
 closes #86

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-undef
```
Comment (HealthStack01) [inline .eslintrc.js:1]: Avoid any changes in this file

File: src/pages/app/clients/Appointments/AppointmentList.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import FilterMenu from '../../../../components/utilities/FilterMenu';
 import { TableMenu } from '../../../../ui/styled/global';
 import { PageWrapper } from '../../styles';
 
+
```
Comment (HealthStack01) [inline src/pages/app/clients/Appointments/AppointmentList.tsx:7]: Change not necessary, or is it?

File: src/pages/app/clients/Client/index.tsx [hunk 3]
```diff
@@ -21,23 +22,27 @@ const AppClient = () => {
       clientResource: {
         ...resource.clientResource,
         show,
-        selectedClient:
-          selectedClient || resource.clientResource.selectedClient,
+        selectedClient: selectedClient?._id
+          ?selectedClient : resource.clientResource.selectedClient,
```
Comment (HealthStack01) [inline src/pages/app/clients/Client/index.tsx:26]:
```suggestion
        selectedClient: selectedClient?._id
          ? selectedClient : resource.clientResource.selectedClient,
```

File: src/pages/app/finance/BillClient/index.tsx [hunk 4]
```diff
@@ -21,16 +21,16 @@ const AppBillClient = () => {
...
+          ?selectedBillClient : resource.billClientResource.selectedBillClient,
       },
     });
 
   const {
     groupedList: clientBillsSummary,
     submit: handleSubmit,
     setFindQuery,
-  } = useRepository(Models.BILLS, handleNavigation);
+  } = useRepository<any>(Models.BILLS, handleNavigation);
```
Comment (HealthStack01) [inline src/pages/app/finance/BillClient/index.tsx:33]: Can you avoid use of `any`?

File: src/pages/app/generic/DetailView.tsx [hunk 5]
```diff
@@ -2,12 +2,12 @@ import React from 'react';
 
 import Button from '../../../components/buttons/Button';
 import { ButtonGroup } from '../../../ui/styled/global';
-import { PageWrapper } from '../../../ui/styled/styles';
 import {
   FullDetailsWrapper,
   GrayWrapper,
   GridWrapper,
   HeadWrapper,
+  PageWrapper,
```
Comment (HealthStack01) [inline src/pages/app/generic/DetailView.tsx:10]: Is change necessary?

File: src/pages/app/generic/FormView.tsx [hunk 6]
```diff
@@ -54,7 +54,7 @@ const FormView: React.FC<Props> = ({
               Create a New {title} by filling out the form below to get started.
               {errors &&
                 JSON.stringify(
-                  ((Object.values(errors)[0] as any) || ({} as any)).message,
```
Comment (HealthStack01) [inline src/pages/app/generic/FormView.tsx:57]: Is change necessary?

**Review by HealthStack01 — CHANGES_REQUESTED** [review]
Good catches.

Resolve requested changes and concerns

File: .eslintrc.js (hunk 1 above)
Comment (inuoluwadunsimi) [reply .eslintrc.js:1]: Noted

File: src/pages/app/generic/FormView.tsx (hunk 6 above)
Comment (inuoluwadunsimi) [reply src/pages/app/generic/FormView.tsx:57]: Oh, not it is not a major change actually, I was testing something there but reverted it back to the original with just spacing changes but I guess it still committed

File: src/pages/app/finance/BillClient/index.tsx (hunk 4 above)
Comment (inuoluwadunsimi) [reply src/pages/app/finance/BillClient/index.tsx:33]: Noted

File: src/pages/app/generic/DetailView.tsx (hunk 5 above)
Comment (inuoluwadunsimi) [reply src/pages/app/generic/DetailView.tsx:10]: Yes actually, I think two different styles with similar names were conflicting so I had to swap it.

File: src/pages/routes.tsx [hunk 7]
```diff
@@ -91,6 +91,7 @@ const AppRoutes = () => {
           />
           <Route path="/app/clients/clients" element={<AppClient />} />
 
+
```
Comment (HealthStack01) [inline src/pages/routes.tsx:94]: 🧹

File: src/ui/styled/styles.ts [hunk 8]
```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 
 export const PageWrapper = styled.div`
   display: flex;
+
```
Comment (HealthStack01) [inline src/ui/styled/styles.ts:5]: Clean up this file, changes unnecessary 🧹

**Review by HealthStack01 — CHANGES_REQUESTED** [review]

**Review by HealthStack01 — APPROVED** [review]

**Pull request merged**